test/unit/org/apache/cassandra/tools/ToolRunner.java

```diff
@@ -136,6 +138,8 @@ public void run()
                     int read = input.read(buffer);
                     if (read == -1)
                     {
+                        if(autoCloseOut)
```

Comment:
Nit: missed whitespace
```suggestion
                        if (autoCloseOut)
```